code/modules/mob/living/simple_animal/vore/badboi.dm

```diff
@@ -0,0 +1,66 @@
+/mob/living/simple_animal/badboi
+	name = "corrupt hound"
+	desc = "Good boy machine broke. At least they haven't stripped the planet down to bare minerals yet."
+	icon = 'icons/mob/vore64x32.dmi'
+	icon_state = "badboi"
+	icon_living = "badboi"
+	icon_dead = "badboi-dead"
+	icon_rest = "badboi_rest"
+	faction = "corrupt"
+	maxHealth = 200
+	health = 200
+	minbodytemp = 150
+	hostile = 1
+	investigates = 1
+	retaliate = 1
```

Comment:
I know where you copied these from uses 1 but be a good pupper and do TRUE for all the bool vars in this file